react/features/overlay/components/OverlayContainer.js

```diff
@@ -1,12 +1,38 @@
-/* global APP */
-
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import PageReloadFilmStripOnlyOverlay from './PageReloadFilmStripOnlyOverlay';
 import PageReloadOverlay from './PageReloadOverlay';
+import SuspendedFilmStripOnlyOverlay from './SuspendedFilmStripOnlyOverlay';
 import SuspendedOverlay from './SuspendedOverlay';
+import UserMediaPermissionsFilmStripOnlyOverlay
+    from './UserMediaPermissionsFilmStripOnlyOverlay';
 import UserMediaPermissionsOverlay from './UserMediaPermissionsOverlay';
 
+declare var APP: Object;
+declare var interfaceConfig: Object;
+
+/**
+ * Maps the overlay component that have to be displayed depending on the
+ * filmstrip only mode.
+ *
+ * @type {object}
+ */
+const filmStripOnlyModeToComponent = {
+    pageReload: {
+        filmStripOnly: PageReloadFilmStripOnlyOverlay,
```

Comment:
I personally don't find the name of the const "filmStripOnlyModeToComponent" clear enough. I also find it more intuitive if we rename it to overlayMode and it's overlayMode[modeKey].suspended, which means store the overlay states by modes, not vice versa. It's a matter of preference though. If we keep it this way let's rename it to "overlayType", this way we'll have overlayType.suspended[componentMapKey];

Adding some jsdoc on what suspended, pageReload and userMediaPermissions mean and reference to would be helpful.